helix-core/src/test/java/org/apache/helix/tools/TestClusterSetup.java

```diff
@@ -355,7 +356,7 @@ public void testDropInstance() throws Exception {
 
     // add fake liveInstance
     ZKHelixDataAccessor accessor =
-        new ZKHelixDataAccessor(clusterName, new ZkBaseDataAccessor<ZNRecord>(_gZkClient));
+        new ZKHelixDataAccessor(clusterName, new ZkBaseDataAccessor<ZNRecord>(ZK_ADDR));
```

Comment:
Question, is this because of the new ZK change? Or other reasons?

Reply:
The method has been deprecated in November 2019. So I think it is almost recent.